dags/lambda_process_espn.py

```diff
@@ -196,4 +207,46 @@ def update_espn_leagues(event, context):
   return {
     'statusCode': update_res['StatusCode'],
     'body': "Test response"  
+  }
+
+
+
+def process_espn_common():
+
+  league_info = {
+    "leagueId": '891817951',
+    "leagueYear": 2023
+  }
+
+  league_data = {
+    'leagueId': '891817951',
+    'leagueYear': 2023
+  }
+
+  for endpoint in common_api_endpoints.keys():
+    view = common_api_endpoints[endpoint]
+
+    header = {}
+    if common_headers.get(endpoint):
+      header = {'x-fantasy-filter': common_headers.get(endpoint)}
+
+    data_endpoint = extract_from_espn_api(league_info, view, header)
+    
+    league_data[endpoint] = data_endpoint
+
+
+    # Data serialization and upload data to S3
+    for key in league_data.keys():
+      if isinstance(league_data[key], pd.DataFrame):
+        league_data[key] = league_data[key].to_json(orient='records')
+    today = date.today().strftime("%b-%d-%Y")
+    
+    filename = "nba-player-stats-" + str(today) + '.json'
```

Comment:
Alternative way with f strings `f"nba-player-stats-{today}.json"` I think today is already string